discordbot/cogs/auditlogs.py

```diff
@@ -0,0 +1,101 @@
+import discord
+from discord.ext import commands
+
+intents = discord.Intents.default()
+intents.guilds = True
+intents.messages = True
```

Comment:
This doesn't seem to do anything here, you can set intents in main.py before the bot is initialized

Reply:
removed